src/components/POIForm.component.js

```diff
@@ -1,8 +1,189 @@
+    this.onChangeName = this.onChangeName.bind(this)
+    this.onChangeDescription = this.onChangeDescription.bind(this)
+    this.onChangeLinks = this.onChangeLinks.bind(this)
+    /*this.onClickPlus = this.onClickPlus.bind(this)*/
+    this.onSubmit = this.onSubmit.bind(this)
+  }
+
+  onDateSelected() {
+    // call this when date input changes
+  }
+
+  handleChange(date) {
+    this.setState({
+      startDate: date
+    })
+  }
+
+  onChangeName(inputName) {
+    this.setState({
+      name: inputName.target.value
+    })
+  }
+
+  onChangeDescription(inputDesription) {
+    this.setState({
+      description: inputDesription.target.value
+    })
+  }
+
+  /*onClickPlus() {
+    this.setState({
+      links : this.state.links.concat(
+        ["1"]
+      )
+    });
+  }*/
+
+  onChangeLinks(inputLinks) {
+    this.setState({
+      links: inputLinks.target.value
+    })
+  }
+
+  onSubmit() {}
+
   render() {
-    return <div>POIForm</div>
+    return (
+      <Form horizontal>
+        <PageHeader>
+          POI Input Form
+          <small>Complete each component below</small>
+        </PageHeader>
+
+        {/*
+          *********THIS IS MY POI NAME***********
+        */}
+        <FormGroup controlId="name">
+          <Col componentClass={ControlLabel} sm={2}>
+            POI Name
+          </Col>
+          <Col sm={10}>
+            <FormControl
+              type="text"
+              placeholder="Enter your POI name here"
+              value={this.state.name}
+              onChange={this.onChangeName}
+            />
+          </Col>
+        </FormGroup>
+
+        {/*
+          *********THIS IS MY DATEPICKER***********
+        */}
+        <FormGroup>
+          <Col componentClass={ControlLabel} sm={2}>
+            POI Date
+          </Col>
+          <Col sm={6}>
+            <DatePicker
+              selected={this.state.startDate}
+              onChange={this.handleChange}
+            />
+          </Col>
+        </FormGroup>
+
+        {/*
+          *********THIS IS MY DESCRIPTION AREA***********
+        */}
+        <FormGroup controlID="description">
+          <Col componentClass={ControlLabel} sm={2}>
+            POI description
+          </Col>
+          <Col sm={10}>
+            <FormControl
+              componentClass="textarea"
+              placeholder="Enter you POI description here"
+              value={this.state.description}
+              onChange={this.onChangeDescription}
+            />
+          </Col>
+        </FormGroup>
+
+        {/*
+          *********THIS IS MY UPLOAD BUTTON***********
+        */}
+        <FormGroup controlID="chooseFile">
+          <Col componentClass={ControlLabel} sm={2}>
+            Upload Media
+          </Col>
+          <Col sm={10}>
+            <FormControl type="file" placeholder="Upload your files here" />
+          </Col>
+        </FormGroup>
+
+        {/*
+          *********THIS IS MY LINKS TEXT AREA***********
```

Comment:
I'll do some more research on the best way to implement this part. We can worry about that in a later PR

Reply:
we found a react-bootstrap-table thing that might work to manage they links that the user will input: http://allenfang.github.io/react-bootstrap-table/example.html#manipulation